datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/DatalakeRestoreFailureReason.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.datalake.flow.dr.restore;
+
+public enum DatalakeRestoreFailureReason {
+    RESTORE_ON_UPGRADE("Restore performed on upgrade"),
```

Comment:
Can you change it to "RESTORE_ON_UPGRADE_FAILURE"?
Restore is not performed on upgrade it is performed to recover from upgrade failure.

Reply:
fixed. thx!